OrbitQt/orbitmainwindow.cpp

```diff
@@ -631,15 +644,61 @@ void OrbitMainWindow::OpenCapture(const std::string& filepath) {
   ui->MainTabWidget->setCurrentWidget(ui->CaptureTab);
 }
 
-void OrbitMainWindow::OpenDisassembly(std::string a_String, DisassemblyReport report) {
-  auto* dialog = new OrbitDisassemblyDialog(this);
-  dialog->SetText(std::move(a_String));
-  dialog->SetDisassemblyReport(std::move(report));
-  dialog->setWindowTitle("Orbit Disassembly");
-  dialog->setAttribute(Qt::WA_DeleteOnClose);
-  dialog->setWindowFlags(dialog->windowFlags() | Qt::WindowMinimizeButtonHint |
-                         Qt::WindowMaximizeButtonHint);
-  dialog->show();
+static std::pair<QString, QString> SplitLineAtFirstLinebreak(const std::string& input) {
```

Comment:
Let us take this moment to actually change the way we store the disassambly/source-code, such that it has a title and a body/code part.

Reply:
Then you also don't need to do all that shifting with the line numbers. 
If you wish, I can also do this as a follow-up pr, but shouldn't be to hard to achieve.

Reply:
completely removed! thanks!